src/java.base/share/classes/com/sun/crypto/provider/AESCipher.java

```diff
@@ -58,13 +58,14 @@
  * @see OutputFeedback
  */
 
-abstract class AESCipher extends CipherSpi {
+class AESCipher extends CipherSpi {
```

Comment:
If you're doing general cleaning, line 33 import ByteBuffer can go, and there's an extra line in the javadoc at line 53 that could also go.

Reply:
Yes. I will make the cleanup as well.